languages.toml

```diff
@@ -3726,3 +3727,29 @@ grammar = "typescript"
 "{" = "}"
 "(" = ")"
 '"' = '"'
+
+
+[[language]]
+name = "spade"
+scope = "source.spade"
+roots = ["swim.toml"]
+file-types = ['spade']
+injection-regex = "spade"
+comment-tokens = ["//", "///", "//"]
+block-comment-tokens = [
+  { start = "/*", end = "*/" },
+  { start = "/**", end = "*/" },
+]
+language-servers = [ "spade-language-server" ]
+indent = { tab-width = 4, unit = "    " }
+
+[language.auto-pairs] 
+'(' = ')' 
+'{' = '}' 
+'[' = ']' 
+'"' = '"' 
+'<' = '>' 
```

Comment:
This doesn't seem to work and I can't figure out why, is it something wrong with the config?

Reply:
This was caused by me using my local config which overrode the one I tried adding to the runtime, should be fixed now